app/forms/assessor_interface/verify_references_form.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class AssessorInterface::VerifyReferencesForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  attr_accessor :application_form
+  attribute :verify_references, :boolean
+  validates :verify_references, inclusion: [true, false]
+
+  delegate :assessment, to: :application_form
+
+  def save
+    return false if invalid?
+
+    if verify_references == true
+      application_form.update!(references_verified: true)
+    end
```

Comment:
```suggestion
    application_form.update!(references_verified: verify_references)
```

Do we need to allow assessors to say "No" after having previously said "Yes"?